### PR TITLE
COGS Integration

### LINKED ISSUE #24: Use RDFTab for extracting terms for imports
Implement MIREOT extract using SQLite/RDFTab

### PR DESCRIPTION
**Pull request by beckyjackson**
### COGS Integration (WIP!)

This currently requires `credentials.json` in the same directory as the `Makefile`, but we should probably find a better way to store / retrieve this? Also the `EMAIL` var is kind of a workaround right now, but it's required.

To set up:
```
make EMAIL=myemail@domain.com init
```

This will create a new Google Sheet named "ONTIE [branch name]", add all the template sheets, and display the URL.

After making any changes in that sheet:
```
make update-cogs
```

Once that's complete, check out the sheet again to see any problems.

### Add fast extraction

Resolves #24

Extract annotations on all terms in `external.tsv` from DOID and OBI using SQLite. Currently excluding NCBITaxon, as this takes a very long time to download, but we can easily add more imports in the future. There is also one term from GO in `external.tsv`, but that doesn't seem like enough to justify downloading GO?

Right now, I have it extracting the following annotations:
* definition
* definition source
* example of usage
* alternative term

We're missing some DOID synonyms, as they use the `oboInOwl` properties. Do we want to include these? We could also include them as "alternative terms" instead.

### CODE REVIEW

File: src/ontology/templates/other.tsv [hunk 1]
```diff
@@ -23,3 +24,978 @@ other disease course	disease course	A disease course that cannot be described by
 transplantation or transfusion	administering substance in vivo	Transfering a solid tissue (transplant) or bodily fluid (transfusion) to an organism.	The process of surgically attaching a donor kidney into a recipient patient.	We intend to map this term to OBI:0000105.
 unknown	information content entity	An organism's lifespan for which there is no available information on an exposure to a material entity of interest.	The lack of information regarding if a human has ever lived in or visted an area where Dengue virus is common.
 no exposure	information content entity	An organism's lifspan which does not include exposure to a substance of interest.	The information that a human has never lived in nor visted an area where Dengue virus is common.
+
```
Comment (jamesaoverton) [inline src/ontology/templates/other.tsv]: Please clean up all these blank lines, when you get a minute.

**Comment by jamesaoverton** [comment]
This all worked for me. The only significant change I made was marking tasks as `.PHONY` in the `Makefile` as appropriate.

**Comment by beckyjackson** [comment]
How do we want to handle the Google API credentials & user email? I think that's the last thing we need to decide before this is ready for review.

**Comment by jamesaoverton** [comment]
I'm going to review this and figure out those last few things.

**Pull request merged**